apps/core/tailwind.config.ts

```diff
@@ -87,6 +87,8 @@ export default {
                 130: '1.3',
             },
             boxShadow: {
+                default:
```

Comment:
Is this really what we call it in the figma? Can we change this name?

Reply:
@mystie711 can you confirm. I believed we agreed it is `default`

Reply:
Before I go and rename it everywhere in this PR, what should we call this one. I have consulted with ChatGPT and it tells me to use `MistyEdge`, can we go with this @mystie711 ?

<img width="714" alt="Screenshot 2023-04-28 at 10 27 37 PM" src="https://user-images.githubusercontent.com/127577476/235285360-3f78f3af-46e0-4185-8efe-ed4d59349af0.png">

CC: @Jordan-Mysten

Reply:
alright. :)